shared/featureconfig/config.go

```diff
@@ -53,6 +53,7 @@ type Flags struct {
 	EnableInitSyncQueue                        bool   // EnableInitSyncQueue enables the new initial sync implementation.
 	EnableFieldTrie                            bool   // EnableFieldTrie enables the state from using field specific tries when computing the root.
 	EnableBlockHTR                             bool   // EnableBlockHTR enables custom hashing of our beacon blocks.
+	EnableRefCopy                              bool   // EnableRefCopy copies the references to objects  instead of the objects themselves when copying.
```

Comment:
```suggestion
	EnableRefCopy                              bool   // EnableRefCopy copies the references to objects instead of the objects themselves when copying.
```